stytch/version.py

```diff
@@ -1 +1 @@
-__version__ = "9.2.0"
```

Comment:
So maybe it's a _bit_ odd, but this is actually a MINOR bump in Python. Changing a variable from optional -> required would be MAJOR, but required -> optional is MINOR since no existing code breaks.